api/platformpb/platform.proto

```diff
@@ -112,6 +112,19 @@ message OrganizationEntitlement {
   Platform platform = 11;
 }
 
+message GetContactInformationRequest {}
+
+message GetContactInformationResponse {
```

Comment:
kept the structure of the response similar to Potal https://github.com/percona-platform/platform/blob/main/api/org/org_api.proto#L468